docs/sphinx/_ext/sphinx-mdx-builder/sphinxcontrib/mdxbuilder/writers/mdx.py

```diff
@@ -331,8 +333,19 @@ def visit_Text(self, node: nodes.Text) -> None:
         if "This parameter will be removed" in content:
             return
 
+        # Prevents wrapped lines in list items, for example in parameter descriptions:
+        #
+        # Args:
+        #     group_name (Optional[str], optional): The name of the asset group.
+        #     dagster_airbyte_translator (Optional[DagsterAirbyteTranslator], optional): The translator to use
+        #         to convert Airbyte content into :py:class:`dagster.AssetSpec`.
+        #         Defaults to :py:class:`DagsterAirbyteTranslator`.
+        if self.in_list_item:
+            content = content.replace("\n", " ")
+
         if self.in_literal and not self.in_literal_block:
-            content = node.astext().replace("<", "\\<").replace("{", "\\{")
+            content = content.replace("<", "\\<").replace("{", "\\{")
```

Comment:
Fixed an issue where content was being overwritten with `node` text, so that these transformations weren't able to compound.